src/tools/fuzzing/fuzzing.cpp

```diff
@@ -2323,6 +2323,13 @@ Expression* TranslateToFuzzReader::makeCompoundRef(Type type) {
   // least avoids infinite recursion here, and we emit a valid (but not that
   // useful) wasm.
   if (type.isNonNullable() && (random.finished() || nesting >= LIMIT)) {
+    // If we have a function context then we can at least emit a local.get,
```

Comment:
It would be good to move some of this info into the comment above, which currently makes it sound like we will definitely trap at runtime.

Reply:
Good idea, I updated the comment above.